player/command.c

```diff
@@ -652,8 +652,8 @@ static int mp_property_avsync(void *ctx, struct m_property *prop,
     if (!mpctx->ao_chain || !mpctx->vo_chain)
```

Comment:
was mp_property_av_speed_correction affected too?

Reply:
Those are speed factors, they never really go that low to trigger scientific notation. Do you want me to add the truncate also here?

Reply:
Should be fine then